esx_service/utils/vmdk_utils.py

```diff
@@ -64,6 +64,9 @@
 VMDK_RETRY_COUNT = 5
 VMDK_RETRY_SLEEP = 1
 
+# root for all the volumes
+VOLUME_ROOT = "/vmfs/volumes/"
```

Comment:
If you introduce this (which is good), you'd have to update the code to use it, at east in vmdk_utils.py